ipn/ipnlocal/local_test.go

```diff
@@ -1625,28 +1625,40 @@ func TestApplySysPolicy(t *testing.T) {
 		{
 			name: "prefs set without policies",
 			prefs: ipn.Prefs{
-				ControlURL:  "1",
-				ShieldsUp:   true,
-				ForceDaemon: true,
+				ControlURL:             "1",
+				ShieldsUp:              true,
+				ForceDaemon:            true,
+				ExitNodeAllowLANAccess: true,
+				CorpDNS:                true,
+				RouteAll:               true,
 			},
 			wantPrefs: ipn.Prefs{
-				ControlURL:  "1",
-				ShieldsUp:   true,
-				ForceDaemon: true,
+				ControlURL:             "1",
+				ShieldsUp:              true,
+				ForceDaemon:            true,
+				ExitNodeAllowLANAccess: true,
+				CorpDNS:                true,
+				RouteAll:               true,
 			},
 		},
 		{
 			name: "empty prefs with policies",
 			wantPrefs: ipn.Prefs{
-				ControlURL:  "1",
-				ShieldsUp:   true,
-				ForceDaemon: true,
+				ControlURL:             "1",
+				ShieldsUp:              true,
+				ForceDaemon:            true,
+				ExitNodeAllowLANAccess: true,
+				CorpDNS:                true,
+				RouteAll:               true,
 			},
 			wantAnyChange: true,
 			stringPolicies: map[syspolicy.Key]string{
 				syspolicy.ControlURL:                "1",
 				syspolicy.EnableIncomingConnections: "never",
 				syspolicy.EnableServerMode:          "always",
+				syspolicy.ExitNodeAllowLANAccess:    "always",
+				syspolicy.EnableTailscaleDNS:        "always",
```

Comment:
not sure if needed - but how about the test case when reading the policy returns an error or blank default value?

Reply:
Added more thorough tests for each policy item (including new ones that are added in future without test changes being necessary) in `TestPreferencePolicyInfo`.